src/BIMDataComponents/BIMDataSelect/BIMDataSelectMulti.vue

```diff
@@ -94,17 +108,40 @@ export default {
       type: Boolean,
       default: false,
     },
+    search: {
+      type: Boolean,
+      default: false,
+    },
+    searchPlaceholder: {
+      type: String,
+      default: "Search",
```

Comment:
I would rather let an empty string as default value for `searchPlaceholder` to avoid i18n issues.